packages/cozy-client/src/CozyClient.js

```diff
@@ -36,6 +36,7 @@ import zip from 'lodash/zip'
 import forEach from 'lodash/forEach'
 import get from 'lodash/get'
 import MicroEE from 'microee'
+import jsonPackage from '../package.json'
```

Comment:
Nit: Downside here is that the whole package.json is included in the bundle where we only need the version.

I thought of using babel to do this, the search and replace plugin would work I think.

https://www.npmjs.com/package/babel-plugin-search-and-replace

Something along the line of

```
CozyClient.version = typeof PACKAGE_JSON_VERSION === 'undefined' ? '0.0.0' : PACKAGE_JSON_VERSION
```

with

babel.config.js

```
import package from './package.json

{
  ...,
  "plugins": [
    [
      "search-and-replace",
      {
        "rules": [
          {
            "search": "PACKAGE_JSON_VERSION",
            "replace": package.version
          }
        ]
      }
     ]
   ]
}
```

Reply:
Good Idea 💡